scripts/generate_report.sh

```
@@ -34,35 +40,64 @@ log "- From: $FROM_DATE"
-        commitContributionsByRepository {
-          repository {
+  }
+' -f user="$USERNAME")
+
+ORGS=$(echo "$ORGS_QUERY" | jq -r '.data.user.organizations.nodes[].login')
+if [ -z "$ORGS" ]; then
+  log "No organizations found for user $USERNAME."
+  ORGS=()  # 空配列
+fi
+
+log "Found Organizations: $(echo "$ORGS" | xargs)"
```

Shellcheck (0.10.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)